### ISSUE TITLE
Inconsistent labels in docker images

### ISSUE THREAD

**Issue by liamchampton** [eclipse/codewind#1271]
**Codewind version:** latest
**OS:** macOS

**Description:**
The labels in the docker images are not consistent. PFE has some good labels, whereas performance doesn't. See output below.

PFE:
```
"Labels": {
                "org.label-schema.build-date": "2019-11-25T15:53:15+0000",
                "org.label-schema.description": "Codewind PFE",
                "org.label-schema.license": "GPLv2",
                "org.label-schema.name": "Codewind PFE",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.url": "https://codewind.dev/",
                "org.label-schema.vcs-ref": "d4d52ca990b720ebfd9929ca81744f68e550ebb6",
                "org.label-schema.vcs-url": "https://github.com/eclipse/codewind",
                "org.label-schema.vendor": "IBM",
                "org.label-schema.version": ""
            }
```

Performance:

```
"Labels": {
                "org.label-schema.build-date": "2019-11-25T16:00:13+0000",
                "org.label-schema.vcs-ref": "d4d52ca990b720ebfd9929ca81744f68e550ebb6",
                "org.label-schema.version": ""
            }
```

Having useful labels within all the docker images could help us better search for images and do system clean up with the CLI. This could be a better alternative to the current solution of using image names. 

Example: There could be an instance where we would want to do a docker system prune but dont want to interfere with other images on the hosts system. Having a label could help us filter out codewind images to prune. See docker docs https://docs.docker.com/engine/reference/commandline/system_prune/

**Steps to reproduce:**
1. Download the latest or a release version set of images
2. Docker inspect the images and look at the labels

**Comment by jopit** [comment]
Also, we should clean up `"org.label-schema.license": "GPLv2"` and `"org.label-schema.vendor": "IBM"` in the PFE labels

**Comment by jopit** [comment]
Seems the label schema we're using (http://label-schema.org/rc1/) has been deprecated in favour of the OCI IMAGE SPEC (https://github.com/opencontainers/image-spec), specifically annotations (https://github.com/opencontainers/image-spec/blob/master/annotations.md)

**Comment by NikCanvin** [comment]
should be: "org.label-schema.license": "EPL-2.0"

**Comment by liamchampton** [comment]
We don't explicitly write this label, it is being inherited from the `centos:7` base image PFE is using. This is the output from centos/where it seems to be coming from:
```
"Labels": {
                "org.label-schema.build-date": "20190305",
                "org.label-schema.license": "GPLv2",
                "org.label-schema.name": "CentOS Base Image",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "CentOS"
            }
```

**Comment by NikCanvin** [comment]
OK - in which case I see no issue leaving it as GPLv2 in this case
(it's not shipped by the Eclipse community servers directly, so it does no impact the Eclipse Codewind projects EPL-2.0 license declaration)

**Comment by liamchampton** [comment]
Fixed in 0.7.0:
eclipse/codewind-pfe:
```
"Labels": {
                "org.label-schema.build-date": "20190927",
                "org.label-schema.license": "GPLv2",
                "org.label-schema.name": "CentOS Base Image",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "CentOS",
                "org.opencontainers.image.created": "2019-12-13T11:06:05+0000",
                "org.opencontainers.image.description": "Codewind PFE",
                "org.opencontainers.image.revision": "65680d72740d77b8a23b013d7db9f170e70ab496",
                "org.opencontainers.image.source": "https://github.com/eclipse/codewind",
                "org.opencontainers.image.title": "Codewind-PFE",
                "org.opencontainers.image.url": "https://codewind.dev/",
                "org.opencontainers.image.version": ""
            }
```

eclipse/codewind-performance:
```
"Labels": {
                "org.opencontainers.image.created": "2019-12-13T11:08:49+0000",
                "org.opencontainers.image.description": "Codewind Performance",
                "org.opencontainers.image.revision": "65680d72740d77b8a23b013d7db9f170e70ab496",
                "org.opencontainers.image.source": "https://github.com/eclipse/codewind",
                "org.opencontainers.image.title": "Codewind-Performance",
                "org.opencontainers.image.url": "https://codewind.dev/",
                "org.opencontainers.image.version": ""
            }
```
